content/articles/account-subscription-balance.md

```diff
@@ -6,39 +6,43 @@ categories:
 ---
 
 # Account Balance
```

Comment:
Wanted to check that we were all on the same page re: changing this title to **Understanding Your Account Balance and Billing Details**

Reply:
LGTM. I like the title change.